<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Regist_Member</class>
 <widget class="QDialog" name="Regist_Member">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>683</width>
    <height>370</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
     <weight>75</weight>
     <bold>true</bold>
     <underline>true</underline>
    </font>
   </property>
   <property name="text">
    <string>신규 성도 등록</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>63</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>* 등    록    일:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>254</x>
     <y>61</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>* 성           명 :</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>252</x>
     <y>83</y>
     <width>131</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>* 성명(동명구분)</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>13</x>
     <y>87</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>* 개  별  코  드:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>257</x>
     <y>110</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>* 합  산  코  드</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>22</x>
     <y>165</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>비            고</string>
   </property>
  </widget>
  <widget class="QDateEdit" name="reg_date_widget">
   <property name="geometry">
    <rect>
     <x>131</x>
     <y>60</y>
     <width>121</width>
     <height>22</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_diff_widget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>85</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="name_widget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>60</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="code1_widget">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>87</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="hap_code_widget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>110</y>
     <width>113</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLineEdit" name="marks_widget">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>160</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>23</x>
     <y>112</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>직            분</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="position_widget">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>110</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>0</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>* 합산코드는 성도검색을 통해서 확인하세요</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>183</y>
     <width>391</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>* 가 있는 항은 필수 사항 이므로 반드시 입력하세요</string>
   </property>
  </widget>
  <widget class="QPushButton" name="add_mem_Button">
   <property name="geometry">
    <rect>
     <x>524</x>
     <y>160</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>추가하기(F12)</string>
   </property>
   <property name="shortcut">
    <string>F12</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>36</y>
     <width>661</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>591</width>
     <height>0</height>
    </size>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>651</width>
     <height>16</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>16</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Plain</enum>
   </property>
   <property name="lineWidth">
    <number>3</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QTableWidget" name="regist_member_tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>220</y>
     <width>641</width>
     <height>141</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="focusPolicy">
    <enum>Qt::NoFocus</enum>
   </property>
   <column>
    <property name="text">
     <string>등록일</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>개별코드</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>성명(동명구분)</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>성명</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>합산코드</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>직분</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>주소</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>비고</string>
    </property>
   </column>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>24</x>
     <y>138</y>
     <width>111</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>주            소</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="addr_widget">
   <property name="geometry">
    <rect>
     <x>132</x>
     <y>137</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
  </widget>
  <widget class="QPushButton" name="member_serch_Button">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>112</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>성도검색</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>494</x>
     <y>61</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>* 합산할 가족이 없으면</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_16">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>20</y>
     <width>291</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>* 합산코드는 합산할 성도의 개별코드입니다.</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_17">
   <property name="geometry">
    <rect>
     <x>504</x>
     <y>80</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>개별코드와 동일합니다.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="member_serch_Button_2">
   <property name="geometry">
    <rect>
     <x>580</x>
     <y>3</y>
     <width>61</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>종료</string>
   </property>
  </widget>
 </widget>
 <tabstops>
  <tabstop>reg_date_widget</tabstop>
  <tabstop>name_widget</tabstop>
  <tabstop>name_diff_widget</tabstop>
  <tabstop>position_widget</tabstop>
  <tabstop>hap_code_widget</tabstop>
  <tabstop>addr_widget</tabstop>
  <tabstop>marks_widget</tabstop>
  <tabstop>add_mem_Button</tabstop>
  <tabstop>code1_widget</tabstop>
  <tabstop>regist_member_tableWidget</tabstop>
  <tabstop>member_serch_Button</tabstop>
  <tabstop>member_serch_Button_2</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>add_mem_Button</sender>
   <signal>clicked()</signal>
   <receiver>Regist_Member</receiver>
   <slot>member_append()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>558</x>
     <y>167</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>230</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>member_serch_Button</sender>
   <signal>clicked()</signal>
   <receiver>Regist_Member</receiver>
   <slot>member_serch()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>519</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>428</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>member_serch_Button_2</sender>
   <signal>clicked()</signal>
   <receiver>Regist_Member</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>575</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>605</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>member_append()</slot>
  <slot>remove_row()</slot>
  <slot>member_save()</slot>
  <slot>save_cancel()</slot>
  <slot>member_serch()</slot>
 </slots>
</ui>
